broker/test/test_broker/iaas.AzureClient.spec.js

```diff
@@ -475,6 +475,7 @@ describe('iaas', function () {
         });
         return client.remove(`${blobName}`)
           .then(result => {
+            console.log(result);
```

Comment:
do u want to keep it ? It should be fine since it is in the test. just checking if this is by mistake or intentional

Reply:
Missed it. Will remove.